spec/fixtures/metadata.json

```diff
@@ -22,7 +22,6 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
```

Comment:
redhat 7 is EoL and got removed from facterdb 3.

Reply:
oh, shame.  One release that included RHEL7 without legacy facts could have provided a useful tool to those still working on their upgrades.  After all, Puppet Inc/Perforce are still shipping and supporting Puppet 8 packages on RHEL 7.

Reply:
and it's _almost_ EOL.  Still not actually for another couple of weeks.

Reply:
yes. but we agrees on it at https://voxpupuli.org/blog/2024/05/08/CentOS-7-CentOS-Stream-8-deprecation/ . And it will take some time until we can roll it out.